mgr/Construct

```diff
@@ -278,7 +278,13 @@ if ($STAR_SYS =~ /x86_darwin/) {
 #$items .= "StBTofPool|StFgtPool|StPeCMaker";
 # closing exclusion list
 my $items = "";# print "items $#items  = @items\n";
-if ( defined($SKIP_DIRS) ){ $items = join("|",split(" ",$SKIP_DIRS));}
+
+if ( defined($SKIP_DIRS) ) {
+  $items = join("|", split(" ", $SKIP_DIRS));
+} else {
+  $items = "g2r|GeoTestMaker|Kinematics|StEEmcPool|StFgtPool|StHighptPool|StJetFinder|StShadowMaker|StSpinMaker|StHbtMaker";
```

Comment:
```suggestion
  push @items, qw(
    g2r
    GeoTestMaker
    Kinematics
    StEEmcPool
    StFgtPool
    StHighptPool
    StJetFinder
    StShadowMaker
    StSpinMaker
    StHbtMaker
  );
```

Reply:
Isn't $items supposed to be a string?

https://github.com/star-bnl/star-sw/blob/70ee73d7d7804f75a4c98f548c4984d0b45a2cf9/mgr/Construct#L280-L286

Reply:
Array variables are referenced as `@items`, I suppose they live in a separate namespace.

Reply:
It is super confusing when the same name is used for two different types in the same context... In support of my implementation I can only say that it seems to work as it passes the CI build as expected without needing to specify the %-patterns.